hal/halx86/generic/halinit.c

```diff
@@ -54,8 +54,8 @@ HalInitializeProcessor(
     KeGetPcr()->StallScaleFactor = INITIAL_STALL_COUNT;
 
     /* Update the interrupt affinity and processor mask */
-    InterlockedBitTestAndSet((PLONG)&HalpActiveProcessors, ProcessorNumber);
-    InterlockedBitTestAndSet((PLONG)&HalpDefaultInterruptAffinity,
+    InterlockedBitTestAndSetAffinity(&HalpActiveProcessors, ProcessorNumber);
+    InterlockedBitTestAndSetAffinity(&HalpDefaultInterruptAffinity,
                              ProcessorNumber);
```

Comment:
```suggestion
                                     ProcessorNumber);
```
or putting it on the same line as the rest of the function

Reply:
fixed